### ISSUE TITLE
FYI: Selfmade dipole antenna

### ISSUE THREAD

**Issue by rgr101** [rstrouse/ESPSomfy-RTS#297]
The transmission power is already very good with the small stub antenna. 
To increase the reception power I built a dipole antenna. It works really well, the performance is noticeably stronger and now reaches all 3 floors of our house.

See picture att.: I got the SMA connector from a disused stub antenna; the dipole wires are cut to exactly 17.3 cm each.

But be careful: Never change the antenna during operation (I made bad experiences with this myself...)!

![IMG_0259](https://github.com/rstrouse/ESPSomfy-RTS/assets/2647778/2e9b1b92-e8f9-4925-8658-cd1b34e16d8d)

**Comment by rstrouse** [comment]
Yeah I have fried a device attempting to shield the transceiver while in operation.   How much better is the remote reception with this?  In 2.4.0 I added the ability to repeat remotes to relay to a central ESPSomfy device.  Not a mesh but a method to localize a receiver.

**Comment by rgr101** [comment]
I can't quantify this exactly now, but the improvement is clearly noticeable. I placed the ESP in the middle of the house, and with the stub antenna it could not always reliably "hear" some remote controls from other floors that were far away. With the dipole, on the other hand, the hit rate is almost 100%.
However, I also optimized the power supply at the same time, so I can't say for sure whether the improvement is solely due to the dipole.

If you would like a more precise statement, I will be happy to make a comparative series in the next few days.

**Comment by Jezza34000** [comment]
Amazing ! I have to try this !
Thank you @rgr101

**Comment by Jezza34000** [comment]
So I tried to build the dipole antenna and I must admit that I am disappointed...
For my side I obtain much better results with a simple straight wire of 69.24cm...

Bench test (remote are wall mounted) :
Simple Wire of 69,24 cm (Remote 1 : -51 ~ -54 Remote  2 : -45 ~ -50 Remote 3 : -65 ~ -67)
Dipole Antenna =  (Remote 1 : -57 ~ -65 Remote  2 : -61~ -69 Remote 3 : -77~ -80)

**Comment by rgr101** [comment]
@Jezza34000 
Hi there, 
Oh, I'm so sorry about that! 
I can only say that the dipole works well for me, but I find your experience with the one-lambda monopole very interesting and will certainly come back to it when I get into trouble with my dipole...
The ESP receives my most distant remote control at -63 to -71 dBi, so in a similar range as yours. But I'm still happy with that, because before (with the standard stub antenna) the measurements were only between -70 and -85 and a lot of dropouts, so now I have a significant improvement. Of course - the better is the enemy of the good, and I would also be happier with a range in the 40/50s...... :)

I'm not a radio or antenna expert, and I get the feeling that there aren't many of them - especially in the "fancy smarthome" world... Everything I hear or read about antenna selection and positioning so far is try, try, try - not exactly the preferred approach for an engineer...

How did you position the antenna? Horizontal or vertical? My problem is that I have to bridge 3 floors. I have the ESP more or less in the middle of the house on the middle floor, and the remotes that are furthest away have to go through at least one thick and strong reinforced concrete ceiling and one or more brick walls. 
I have made the best experience with a horizontal orientation, as the antenna reaches the different floors better that way. However, the horizontal propagation in the area is not so good, but at least it works... As far as I know, with the monopole, and especially with the 1-lambda, there is a shraper propagation at the ends of the antenna, which would probably lead to more problems for me. The lambda/4 dipole is considered a balanced middle ground here. But as I said, I am anything but an antenna expert 😇
